src/llm/mcp_servers/services/job_search_service.py

```
@@ -0,0 +1,134 @@
+        glassdoor_jobs = await self._run_scraper_with_filtering(
+            'glassdoor', self.glasdoor_jobs_scraper_service, job_title, location, remote, user_id, forbidden_titles)
+        if glassdoor_jobs:
+            jobs.extend(glassdoor_jobs)
+
+        return jobs
+    
+    async def get_jobs_from_linkedin(self, job_title: Optional[str] = None, remote: Optional[bool] = None,
+                                   user_id: Optional[str] = None) -> List:
+        """Search for jobs on LinkedIn"""
+        job_title, location, remote, forbidden_titles = await self._get_search_params_from_config_or_default(
+            job_title, location, remote)
```

Ruff (0.14.13)

Undefined name `location`

(F821)